storehaus-memcache/src/main/scala/com/twitter/storehaus/memcache/MergeableMemcacheStore.scala

```diff
@@ -77,7 +77,7 @@ class MergeableMemcacheStore[K, V](underlying: MemcacheStore, maxRetries: Int)(k
               inj.invert(cbValue) match {
                 case Success(v) => // attempt cas
                   val resV = semigroup.plus(v, kv._2)
-                  underlying.client.cas(key, inj.apply(resV), casunique).flatMap { success =>
+                  underlying.client.cas(key, 0, underlying.ttl.fromNow, inj.apply(resV), casunique).flatMap { success =>
```

Comment:
Found an explanation here
https://github.com/memcached/memcached/blob/master/doc/protocol.txt#L155
and here
http://stackoverflow.com/questions/12228097/what-is-the-flags-arg-in-finagle-memcached-set-method

The flag is opaque to the server and is returned back along with the value when reading. For correctness, it looks like we should set the same flag as the underlying store does for `set` calls. It's probably been working fine all this while because of the default being 0 in `MemcacheStore`. Not your change, but would be nice to fix this as well. What do you think?